recipes/zenodo-backpack/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "zenodo_backpack" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5843a04b353ae613b474932c2a12d1fbfd529674074e73927ddc131eba7f1da8
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - conda=conda.cli.pip_warning:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - requests
+    - tqdm
+
+test:
```

Please add a call to `pip check` to check for the integrity with of the requirements specification.
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```